sylt-compiler/src/typechecker.rs

```diff
@@ -117,7 +117,7 @@ enum Constraint {
     Container,
     SameContainer(usize),
     Contains(usize),
-    IsContainedBy(usize),
+    Holds(usize),
```

Comment:
Now holds and contains seem like the same thing.

Reply:
If you want to change it back, I'm fine with the old one.